core/src/iosMain/kotlin/com/revenuecat/purchases/kmp/models/Price.ios.kt

```diff
@@ -25,7 +23,7 @@ public actual data class Price(
 
 internal fun RCStoreProduct.toPrice(): Price =
     localizedPriceString().let { localizedPrice ->
-        Price(
+        com.revenuecat.purchases.kmp.models.Price(
```

Comment:
Hmm is it name colliding with something else? Being in the same file, I would assume it would use the `Price` defined here automatically, but not sure...